modules/ding_frontend/ding_frontend.admin.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * Module's admin features.
+ */
+
+/**
+ * Administration form.
+ */
+function ding_frontend_settings($form, &$form_state) {
+  $types = array(
+    'ding_news' => t('News'),
+    'ding_event' => t('Event'),
+  );
+  $form['ding_frontend_fe_promote_types'] = array(
+    '#title' => t('Promote types to Frontpage'),
+    '#type' => 'checkboxes',
+    '#options' => $types,
+    '#default_value' => variable_get('ding_frontend_fe_promote_types', array('ding_event' => 'ding_event')),
```

Comment:
We can't use `node_options_*` because they imported in feautres, so next feature revert will reset them. Current approach consider that local admin will set `ding_frontend_fe_promote_types` one time, and in next update of site and feature revert, `node_options_*` values will be set from `ding_frontend_fe_promote_types` which saved in  database of updated site.